Classes/Event/EventMetadata.php

```diff
@@ -11,46 +11,25 @@
  * source code.
  */
 
-use Neos\Cqrs\Domain\Timestamp;
-use TYPO3\Flow\Annotations as Flow;
-
 /**
- * The MessageMetadata is a container for arbitrary metadata for Commands and Events.
+ * The EventMetadata is a container for arbitrary metadata for Events.
```

Comment:
Not entirely sure about that. What I did last time was create an command identifier in the command dispatcher and store in the metadata, then use that and store it in the metadata of events that emerged from that command as "correlationId". That way it's possible to easily debug which command lead to which (possibly wrong) events.